CONTRIBUTING.md

```diff
@@ -63,12 +63,24 @@ Once merged to the main branch, `po` files, `README.md` with the command line re
 
 You can also contribute to the documentation. It uses [GitHub Markdown Format](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github).
 
-You can propose modifications in 2 ways:
+To contribute to the documentation edit the files in the `/docs/` directory after forking and cloning the repo.
+Each page on the documentation [website](https://canonical-adsys.readthedocs-hosted.com/en/stable/) also includes
```

Comment:
Small question: wouldn't it be better if we extended the link to match the entirety of `[documentation website]` instead of just `[website]`?

Reply:
> Small question: wouldn't it be better if we extended the link to match the entirety of `[documentation website]` instead of just `[website]`? 

Agreed! These contributing guidelines will also need some other improvements in a future pulse. Thanks.

Reply:
fixed in [56da448](https://github.com/ubuntu/adsys/pull/1069/commits/56da44816225a7d595eeb09d0af8a5d9cd7d6521)